### DISCUSSION TITLE
Aboute cache size and its timings

### DISCUSSION THREAD

**Discussion by phantomcraft** [DNSCrypt/dnscrypt-proxy#1897] [Q&A] [answered]
```
cache_size = 4096

cache_min_ttl = 2400
cache_max_ttl = 86400

cache_neg_min_ttl = 60
cache_neg_max_ttl = 600
```

What does 4096 means in **cache_size**?

Are the timings in milliseconds?

**Comment by vp1981** [comment, marked as answer]
Hello phantomcraft.

> What does 4096 means in cache_size?

I think it is "size of cache", i.e. number of domains `dnscrypt-proxy` will keep it "cache".

> Are the timings in milliseconds?

If you are talking about these numbers
```
cache_min_ttl = 2400
cache_max_ttl = 86400

cache_neg_min_ttl = 60
cache_neg_max_ttl = 600
```
these are seconds.

**Reply by phantomcraft** [reply]
@vp1981

Thank you very much for the answer.
